checkers/preferDecodeRune_checker.go

```diff
@@ -0,0 +1,56 @@
+package checkers
+
+import (
+	"go/ast"
+	"go/types"
+
+	"github.com/go-toolsmith/astcast"
+	"github.com/go-toolsmith/typep"
+
+	"github.com/go-critic/go-critic/checkers/internal/astwalk"
+	"github.com/go-critic/go-critic/framework/linter"
+)
+
+func init() {
+	var info linter.CheckerInfo
+	info.Name = "preferDecodeRune"
+	info.Tags = []string{"performance"}
```

Comment:
Please add `"experimental"` tag as all new checkers should stay experimental for at least 1 release; so we can test them while affecting less users with potential bugs.